src/logic/operate.js

```diff
@@ -1,20 +1,20 @@
-import Big from 'big';
+import Big from 'big-js';
 
 const operate = (number1, number2, operation) => {
   const num1 = Big(number1);
   const num2 = Big(number2);
   switch (operation) {
     case '+':
-      return num1.add(num2);
+      return num1.plus(num2).toString();
     case '-':
-      return num1.minus(num2);
+      return num1.minus(num2).toString();
     case 'X':
-      return num1.mul(num2);
-    case '/':
+      return num1.mul(num2).toString();
+    case '÷':
       if (num2 === 0) return "Can't divide by zero";
-      return num1.div(num2);
+      return num1.div(num2).toString();
     case '%':
```

Comment:
i see that you managed division by zero....it still doesn't work😊😊 , Try to debug this .
Try to follow the execution flow between `operate.js` and  `calculate.js`

![Screen Shot 2020-02-13 at 6 26 01 PM](https://user-images.githubusercontent.com/40712459/74461321-a0e66e00-4e8e-11ea-8fa3-4895515b16eb.png)

Reply:
Fixed it, the problem was using === it was comparing an object with a number. Used .eq method on the big.js module